src/js/index.js

```diff
@@ -735,6 +764,21 @@ function showAllSelect() {
 	}
 }
 
+function showAllSelectNetwork() {
```

Comment:
@gertie-sheshe we have missed setting the arrow to active at this point. Its always pointing upwards  i.e `inactive`